BraintreeDropIn/BTCardFormViewController.m

```diff
@@ -618,12 +618,10 @@ - (void)basicTokenization {
             self.navigationItem.rightBarButtonItem = addCardButton;
 
             if (error != nil) {
-                dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
Is this change related to drop in legacy?

Reply:
We already switched to the main thread at line 615 and didn't need to do it again.